UefiPayloadPkg/SPI/mmio.c

```diff
@@ -0,0 +1,37 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+/* Originally imported from linux/include/asm-arm/io.h. This file has changed
+ * substantially since then.
+ */
+
+#include <Include/PiDxe.h>
+#include "mmio.h"
```

Comment:
if you need mmio, just use the MdePkg/Library/BaseIoLibIntrinsic/IoLib.c

Reply:
Removed it